### PR TITLE
Add a section on listeners to Migration Guide

### PR DESCRIPTION
**Pull request by rictic**
This came up in a polyup feature request: https://github.com/PolymerLabs/polyup/issues/20

### CODE REVIEW

File: 1.0/docs/migration.md [hunk 1]
```diff
@@ -633,6 +633,23 @@ in the Developer guide, or use `setAttribute` directly.
 Also, note that the `class` attribute is ignored if it is specified in the `hostAttributes` 
 object.
 
+## Listeners {#listeners}
+
+In 0.5 you could declare event listeners for a custom element as an attribute on the 
+`polymer-element` declaration:
+
+    <polymer-element name='magic-button' on-scroll='{{onScrollHandler}}' on-tap='{{wasTapped}}'>
```
Comment (ebidel) [inline 1.0/docs/migration.md]: double quotes please on html attributes

File: 1.0/docs/migration.md [hunk 2]
```diff
@@ -633,6 +633,23 @@ in the Developer guide, or use `setAttribute` directly.
...
+    </polymer-element>
+
+Now you define these listeners on the listeners object on the prototype:
```
Comment (ebidel) [inline 1.0/docs/migration.md]: on the listeners -> om the `listeners`

**Comment by ebidel** [comment]
Reviewed

File: 1.0/docs/migration.md (hunk 1 above)
Comment (rictic) [inline 1.0/docs/migration.md]: Done

File: 1.0/docs/migration.md (hunk 2 above)
Comment (rictic) [inline 1.0/docs/migration.md]: Done

**Comment by rictic** [comment]
All comments addressed, please take a look

**Comment by ebidel** [comment]
LGTM

**Pull request merged**